examples/chapter1/src/example5.cpp

```diff
@@ -66,16 +65,12 @@ int main() {
                               }
                               vectorPublisher->publish(vector);
                               std::this_thread::sleep_for(std::chrono::milliseconds(5));
-			      std::cout << "t (publisher) thread ID: " << std::this_thread::get_id() << std::endl;
+                              std::cout << "vectorPublisherThread thread ID: " << std::this_thread::get_id() << std::endl;
```

Comment:
same rename the variable name here and then on line 71

Reply:
DOne